lsp-protocol.el

```diff
@@ -31,6 +31,17 @@
 (require 'ht)
 (require 's)
 
+(defun lsp-keyword->symbol (keyword)
+  (intern (substring (symbol-name keyword) 1)))
+
+(defun lsp-keyword->string (keyword)
+  (substring (symbol-name keyword) 1))
+
+
+(defun lsp-use-plists ()
+  ;; change to t and recompile lsp-mode to force the usage of plists
+  nil)
+
 (defmacro lsp-interface (&rest interfaces)
```

Comment:
It does not need to be on this PR, but is it possible to make tests for this kind of macro?
This macro seems to be a little complex and a test could help us evolve in the future easily.

Reply:
I started test suite for lsp-protocol.el